### DISCUSSION TITLE
TP-Link TL-WN722N v2 not working monterey mac pro 5.1

### DISCUSSION THREAD

**Discussion by crosswire80** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#324] [Q&A]
HI there, I installed the correct drivers from your github (Wireless USB OC Big Sur Adapter-V16) and after restart, I see no usb wifi plugin under the icon. I have tried plugging the tplink adapter on various usb ports and nothing. 
Much thanks for any advise.

<img width="155" alt="Screen Shot 2024-01-27 at 6 02 11 PM" src="https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/assets/26411397/8fb535af-0b4a-40c6-99f6-3ba7774665bf">

**Comment by chris1111** [comment]
Mac Pro 5.1 why using the OpenCore version ?

**Reply by crosswire80** [reply]
Because my Mac Pro is old, I had to use open core to upgrade to Monterey and used open core v1.1.0. I do not have built in wifi on this Mac Pro

**Comment by chris1111** [comment]
Ok so You use OCLP and you didn't say it
I have the same adapter and it work without issue
Send me here your entire EFI folder as ZIP file, you can Hide the serial Number if you want

**Reply by crosswire80** [reply]
ok thank you. Not sure how to hide SN etc. you can delete the post after. see attached.much thanks.

DELETE EFI

**Comment by chris1111** [comment]
I don't see anything as a problem but you have several kexts that I never use, what I recommend is to uninstall this version 
Using the Uninstaller

![Screenshot 2024-01-27 at 7 25 50 PM](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/assets/6248794/ab42b446-a8dd-4ad7-9519-7139c6cf3d72)

Reboot macOS



And use the version for Macs here [Wireless-USB-Big-Sur-Adapter](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter)
and don't forget at the end  to accepted `System Software Update `in Preferences Security Allow Realtek semiconductor Corp
Then Reboot
Let me know; hope that work.

**Reply by crosswire80** [reply]
Thank you ! it worked with the new version. Just have to figure out why airdrop is not showing up as its not in the finder settings or window. But the tplink wifi works. Much thanks!
